### ISSUE TITLE
Memory parameter is displayed empty in some ACDCs

### ISSUE THREAD

**Issue by sharad1126** [CMSCompOps/WorkflowWebTools#106]
After submitting an ACDC, the final summary that comes doesn't shows the memory requested in the acdc although it does gets assigned according to the logs. 
<img width="729" alt="screen shot 2018-08-03 at 2 29 22 pm" src="https://user-images.githubusercontent.com/17527468/43643442-94034eea-972b-11e8-8e17-015edac59add.png">

The logs are - 

“pdmvserv_task_BPH-RunIIFall17GS-00021__v1_T_180414_204720_8322”: {“Action”: “acdc”, “Reasons”: [], “ACDCs”: [], “user”: “sagarwal”, “Parameters”: {“BPH-RunIIFall17GS-00021_0/BPH-RunIIFall17GS-00021_0MergeRAWSIMoutput/BPH-RunIIFall17DRPremix-00049_0/BPH-RunIIFall17DRPremix-00049_1/BPH-RunIIFall17DRPremix-00049_1MergeAODSIMoutput/BPH-RunIIFall17MiniAODv2-00019_0”: {“cores”: “”, “xrootd”: “enabled”, “sites”: [“T1_UK_RAL”, “T2_IT_Pisa”, “T2_PL_Warsaw”, “T2_US_UCSD”], “memory”: “”}, “BPH-RunIIFall17GS-00021_0/BPH-RunIIFall17GS-00021_0MergeRAWSIMoutput/BPH-RunIIFall17DRPremix-00049_0/BPH-RunIIFall17DRPremix-00049_1”: {“cores”: “”, “xrootd”: “enabled”, “sites”: [“T1_FR_CCIN2P3”, “T1_IT_CNAF”, “T1_UK_RAL”, “T2_BE_IIHE”, “T2_CH_CERN”, “T2_DE_RWTH”, “T2_FR_IPHC”, “T2_IT_Legnaro”, “T2_IT_Pisa”, “T2_US_Caltech”], **“memory”: “20000"**}, “BPH-RunIIFall17GS-00021_0/BPH-RunIIFall17GS-00021_0MergeRAWSIMoutput”: {“cores”: “”, “sites”: [“T2_CH_CERN”, “T2_ES_CIEMAT”], “memory”: “”}, “BPH-RunIIFall17GS-00021_0/BPH-RunIIFall17GS-00021_0MergeRAWSIMoutput/BPH-RunIIFall17DRPremix-00049_0”: {“cores”: “”, “xrootd”: “enabled”, “sites”: [“T1_ES_PIC”, “T1_RU_JINR”, “T1_UK_RAL”, “T2_CH_CERN”, “T2_FR_IPHC”, “T2_IT_Legnaro”, “T2_IT_Pisa”, “T2_UK_London_Brunel”, “T2_US_Vanderbilt”, “T2_US_Wisconsin”], “memory”: “”}, “BPH-RunIIFall17GS-00021_0/BPH-RunIIFall17GS-00021_0MergeRAWSIMoutput/BPH-RunIIFall17DRPremix-00049_0/BPH-RunIIFall17DRPremix-00049_1/BPH-RunIIFall17DRPremix-00049_1MergeAODSIMoutput/BPH-RunIIFall17MiniAODv2-00019_0/BPH-RunIIFall17MiniAODv2-00019_0MergeMINIAODSIMoutput/BPH-RunIIFall17NanoAOD-00019_0”: {“cores”: “”, “xrootd”: “enabled”, “sites”: [“T2_BE_IIHE”, “T2_IT_Pisa”], “memory”: “”}, “BPH-RunIIFall17GS-00021_0”: {“cores”: “”, “xrootd”: “enabled”, “sites”: [“T1_DE_KIT”, “T1_ES_PIC”, “T1_FR_CCIN2P3”, “T1_IT_CNAF”, “T1_RU_JINR”, “T1_UK_RAL”, “T1_US_FNAL”, “T2_BE_IIHE”, “T2_CH_CERN”, “T2_CH_CSCS”, “T2_DE_DESY”, “T2_DE_RWTH”, “T2_ES_CIEMAT”, “T2_FR_CCIN2P3", “T2_FR_GRIF_IRFU”, “T2_FR_GRIF_LLR”, “T2_FR_IPHC”, “T2_IT_Bari”, “T2_IT_Legnaro”, “T2_IT_Pisa”, “T2_PL_Warsaw”, “T2_UK_London_Brunel”, “T2_UK_London_IC”, “T2_US_Caltech”, “T2_US_Florida”, “T2_US_MIT”, “T2_US_Nebraska”, “T2_US_Purdue”, “T2_US_Vanderbilt”, “T2_US_Wisconsin”], “memory”: “”}}}

Can you please have a look?

**Comment by dabercro** [comment]
Yeah, this particular action summary page has been needing to be fixed for a long time. The whole submission form is sort of horrible ([but stable](https://github.com/CMSCompOps/WorkflowWebTools/blob/master/test/test_workflowwebtools.py#L308)) since I didn't know what I was doing when I wrote it.

The memory shown in the logs (getaction) is the version that Unified really sees.
